core/tests/protractor_desktop/coreEditorAndPlayerFeatures.js

```diff
@@ -323,8 +366,11 @@ describe('Core exploration functionality', function() {
     explorationPlayerPage.expectExplorationToNotBeOver();
     explorationPlayerPage.expectInteractionToMatch(
       'MultipleChoiceInput',
-      [forms.toRichText('option A'), forms.toRichText('option B')]);
-    explorationPlayerPage.submitAnswer('MultipleChoiceInput', 'option B');
+      [forms.toRichText('option B'), forms.toRichText('option D'),
+        forms.toRichText('option E'), forms.toRichText('option G'),
+        forms.toRichText('option H'), forms.toRichText('option I')]);
+    explorationPlayerPage.submitAnswer('MultipleChoiceInput', 'option I');
```

Comment:
Can you also try selecting an answer choice that is incorrect?

Reply:
done